### PR TITLE
Update User Guide to fit new Commands

### PR DESCRIPTION
**Pull request by eesung00**
user guide update for new Commands after the migration

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -8,9 +8,13 @@ title: User Guide
 
 * [Quick Start](#quick-start)
 * [Features](#features)
+    * [Viewing help: `help`](#viewing-help-help)
```
Comment (jolynloh) [inline docs/UserGuide.md]: Maybe `Getting help` will be clearer?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -31,12 +35,19 @@ title: User Guide
...
-
+    * **`help`**: Shows a message explaining the way to access help page.
+   
     * **`list`** : Displays internship application.
 
     * **`add`**`c/Google ct/11111111 d/2022-01-01 e/google@example.com p/Software Engineer` : Adds an internship application on `2022-01-01` to `Google` with contact number `11111111` and email `google@example.com` for the role of `Software Engineer` to **CinternS**.
 
     * **`delete`**`2` : Deletes the 2nd application shown in the current list.
+   
+    * **`find`**`google` : Finds the applications that contain keyword **google** in their company name and position.
```
Comment (jolynloh) [inline docs/UserGuide.md]:
```suggestion
    * **`find`**`google` : Finds the applications that contain keyword **google** in their company name and/or position.
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -31,12 +35,19 @@ title: User Guide
...
+   
+    * **`edit`**`1 e/hr@internship.sg`: Edits the 1st application shown in the current list to update new email address.
```
Comment (jolynloh) [inline docs/UserGuide.md]:
```suggestion
    * **`edit`**`1 e/hr@internship.sg`: Edits the 1st application shown in the current list with new email address.
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -61,6 +72,10 @@ title: User Guide
 
 </div>
 
+### Viewing help: `help`
```
Comment (jolynloh) [inline docs/UserGuide.md:75]: Same suggestion as above

File: docs/UserGuide.md [hunk 5]
```diff
@@ -97,6 +112,43 @@ Format: `delete INDEX`
...
+
+`edit 1 ct/91234567 e/grab@example.com` edits the contact and email of the 1st application to be 91234567 and grab@example.com respectively.
+`edit 2 c/Garena` edits the company of the 2nd application to be Garena.
+
+### Clearing all entries: `clear`
+Clears all entries from CinternS.
+
+Format: `clear`
+
 ### Exiting the program: `exit`
```
Comment (jolynloh) [inline docs/UserGuide.md]:
```suggestion
```

File: docs/UserGuide.md [hunk 6]
```diff
@@ -97,6 +112,43 @@ Format: `delete INDEX`
...
+
+* The search is case-insensitive. e.g `google` will match `Google`
+* The order of the keywords does not matter. e.g. `Tech Micron` will match `Micron Tech`
+* Only the company and position is searched.
+* Only full words will be matched e.g. `ByteDance` will not match `ByteDances`
+* Applications matching at least one keyword will be returned (i.e. `OR` search). e.g. `Google Shopee` will return `Shopee HQ`, `Google Singapore`
+
+Examples:
+* `find Software` returns `Google` and `Wise` application whose position contain software keyword.
+![FindExample](images/FindExample.png)
```
Comment (jolynloh) [inline docs/UserGuide.md:129]: Perhaps the image can be updated to one containing the new displayed Date format?

**Review by jolynloh — COMMENTED** [review]
Gave some comments, other than that lgtm

**Review by jhchee18 — COMMENTED** [review]
LGTM!

**Pull request merged**